js/models/stock.js

```diff
@@ -1,106 +1,120 @@
-    },
-    {
-        "Status": "SUCCESS",
-        "Name": "3M Co",
-        "Symbol": "MMM",
-        "LastPrice": 159.05,
-        "Change": 0.0300000000000011,
-        "ChangePercent": 0.0188655515029563,
-        "Timestamp": "Mon May 4 15:59:00 UTC-04:00 2015",
-        "MSDate": 42128.6659722222,
-        "MarketCap": 100891936050,
-        "Volume": 153090,
-        "ChangeYTD": 164.32,
-        "ChangePercentYTD": -3.20715676728334,
-        "High": 159.85,
-        "Low": 158.12,
-        "Open": 158.12
-    },
-    {
-        "Status": "SUCCESS",
-        "Name": "Twitter Inc",
-        "Symbol": "TWTR",
-        "LastPrice": 37.89,
-        "Change": 0.00999999999999801,
-        "ChangePercent": 0.0263991552270275,
-        "Timestamp": "Mon May 4 15:59:00 UTC-04:00 2015",
-        "MSDate": 42128.6659722222,
-        "MarketCap": 24794496090,
-        "Volume": 1394555,
-        "ChangeYTD": 35.87,
-        "ChangePercentYTD": 5.63144689155284,
-        "High": 38.96,
-        "Low": 37.56,
-        "Open": 38.7
-    },
-    {
-        "Status": "SUCCESS",
-        "Name": "Facebook, Inc.",
-        "Symbol": "FB",
-        "LastPrice": 78.81,
-        "Change": -0.179999999999993,
-        "ChangePercent": -0.227876946448908,
-        "Timestamp": "Mon May 4 15:59:00 UTC-04:00 2015",
-        "MSDate": 42128.6659722222,
-        "MarketCap": 221317079160,
-        "Volume": 839079,
-        "ChangeYTD": 78.02,
-        "ChangePercentYTD": 1.01256088182518,
-        "High": 79.69,
-        "Low": 78.64,
-        "Open": 79.28
-    }
-];
 
 // define your Stock model here
+
+// define your Stock model here
+function Stock(symbol){
+  // this.name = name;
+  this.symbol = symbol;
+  this.numShares = 0;
+  this.data = this.lookupStock();
+  this.name = name;
+  this.price = price;
+
+}
+  // your code to search the API and return a Stock object here
+  // $(document).ready(function(){
+  //   var stocksAPI = "http://dev.markitondemand.com/api/v2/quote/jsonp?symbol=" + symbol
+  //   $.ajax({
+  //     url: stocksAPI,
+  //     type: "get",
+  //     dataType: "jsonp",
+  //     context: this
+  //   }).done(function( response  ){
+  //     $("#stock-name").append(response["Name"]);
+  //     $("#stock-price").append(response["LastPrice"]);
+  //     // StockView.render();
+  //       console.log( response["Open"] )
+  //   }).fail(function(){
+  //     console.log("ajax request not successful")
+  // })
+  // })
+  Stock.prototype = {
+    totalValue: function() {
+      return this.price * this.numShares;
+      },
+    lookupStock: function(){
+        var stocksAPI = "http://dev.markitondemand.com/api/v2/quote/jsonp?symbol=" + this.symbol
+      $.ajax({
+        url: stocksAPI,
+        type: "GET",
+        dataType: "jsonp",
+        context: this
+      }).done(function( response ){
+        $("#stock-name").append(response.Name);
+        price = response.LastPrice;
+        // console.log(response.LastPrice)
```

Comment:
this sort of behavior belongs in the view where your appending information to the dom. Additionally, what does price do here, is it jsut a global, or is this something you want associated with the model